Cargo.toml

```diff
@@ -10,11 +10,31 @@ crate-type = ["cdylib", "lib"]
 name = "pgrx_embed_vectors"
 path = "./src/bin/pgrx_embed.rs"
 
+[[bin]]
+name = "prepare_dataset"
+path = "./crates/stand-alone-test/src/bin/prepare_dataset.rs"
+required-features = ["stand-alone-test"]
+
+[[bin]]
+name = "make_hnsw"
+path = "./crates/stand-alone-test/src/bin/make_hnsw.rs"
+required-features = ["stand-alone-test"]
+
+[[bin]]
+name = "search_hnsw"
+path = "./crates/stand-alone-test/src/bin/search_hnsw.rs"
+required-features = ["stand-alone-test"]
+
```

Comment:
These binaries could be crates in package `stand-alone-test`. There is no need to modify `./Cargo.toml`.

https://doc.rust-lang.org/cargo/reference/cargo-targets.html